rfc3164/machine.go.rl

```diff
@@ -141,6 +165,24 @@ rfc3339 = fulldate >mark 'T' hhmmss timeoffset %set_rfc3339 @err(err_rfc3339);
 # note > this could mean that the we may need to create and to use a labelrange = graph{1,63} here if we want the parser to be stricter.
 hostname = (hostnamerange -- ':') >mark %set_hostname $err(err_hostname);
 
+# Cisco IOS devices sometimes include a "message counter" before the timestamp
+# "<189>237: *Jan  8 19:46:03.295..."
+msgcountval = (digit*) >mark %set_msgcount @err(err_msgcount);
+msgcount = (msgcountval ':' sp*) when { m.msgcount };
+# they can also include a "sequence number" after the message counter
+# "<189>237: 000104: *Jan  8 19:46:03.295..."
+sequenceval = (digit+) >mark %set_sequence @err(err_sequence);
+sequence = (sequenceval ':' sp*) when { m.sequence };
+# and optionally put a hostname before the timestamp
+ciscoHostname = (hostname ':' sp*)? when { m.ciscoHostname };
+# and then they prepend a '*' to the timestamp if there is no NTP sync
+ciscostar = ('*'?) when { m.msgcount || m.sequence || m.ciscoHostname };
```

Comment:
Not top priority, but we may want to capture the presence of the asterisk in a machine field (`withoutNTPSync = true`, by default `false`).